rhino.compute/js/SampleBrepBoolean/index.html

```diff
@@ -0,0 +1,559 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <meta charset="utf-8">
+    <title>rhino3dm Sample - Rhino Logo</title>
+    <style>
+        body {
+            margin: 0px;
+            overflow: hidden;
+            font-size: 30pt;
+            font-family: arial;
+        }
+
+        button {
+            font-size: 20pt;
+        }
+
+        .inner {
+            margin: 3px 0px 3px 0px;
+        }
+
+        #glcanvas {
+            width: 100%;
+            height: 100%;
+        }
+
+        #controls {
+            display: flex;
+            flex-direction: column;
+            overflow: auto;
+            position: absolute;
+            top: 0px;
+            padding: 10px;
+        }
+
+        #loader {
+            border: 5px solid #f3f3f3; /* Light grey */
+            border-top: 5px solid #3d3d3d; /* Grey */
+            border-radius: 50%;
+            width: 40px;
+            height: 40px;
+            animation: spin 1s linear infinite;
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            z-index: 2;
+            display: none;
+        }
+
+        @keyframes spin {
+            0% { transform: rotate(0deg); }
+            100% { transform: rotate(360deg); }
+        }
+
+    </style>
+</head>
+
+<body>
+<div id="loader"></div> 
+<script src="https://cdn.jsdelivr.net/npm/three@0.118.1/build/three.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/three@0.118.1/examples/js/controls/OrbitControls.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/rhino3dm/rhino3dm.min.js"></script> <!-- latest -->
+<script src="https://cdn.jsdelivr.net/npm/compute-rhino3d/compute.rhino3d.min.js"></script> <!-- latest -->
+
+<div id="glcanvas"></div>
+<div id="controls">
+    <button id="load_model" class="inner">Fetch Rhino Logo</button>
+    <button id="load_cutters" class="inner">Fetch Cutters</button>
+    <button id="diff" class="inner">BooleanDifference</button>
+    <button class="inner" onclick="RhinoCompute.getAuthToken(false)">Set Authorization</button>
+</div>
+
+<script>
+    // holds top level variables
+    var RHINOLOGO = {};
+    let logo, cutters, difference;
+
+    // load rhino3dm
+    var Module;
+    rhino3dm().then((m) => {
+      Module = m;
+      console.log("rhino3dm loaded!")
+    });
+
+    init();
+    animate();
+
+    /**
+     * Set up empty GL canvas with a gradient background. The geomtry will
+     * be loaded into the scene through a separate routine.
+     */
+    function init() {
+        RhinoCompute.authToken = RhinoCompute.getAuthToken();
+        RhinoCompute.url = 'https://compute.rhino3d.com/';
```

Comment:
This should be updated to point to localhost (perhaps the compute.rhino3d library should default to localhost itself...). Also, could we standardise the way that these samples are structured to make it easier to see the differences between them? I know I'm picking on a particularly old sample here that contains some helper methods that are no longer necessary...

Reply:
I plan to do a second pass to review the code in all of these. I agree that some of these samples need to be updated.

I do think the compute client libraries should default to localhost.

Reply:
Sounds good!